META.coq.in

```diff
@@ -354,7 +354,7 @@ package "plugins" (
     description = "Coq extraction plugin"
     version     = "8.10"
 
-    requires    = "coq.plugins.ltac"
+    requires    = "coq.plugins.ltac, zarith"
```

Comment:
I am not sure this is accurate, right? As far as I understand the plugin itself doesn't require `zarith` to run, right?

Reply:
The plugin doesn't require `zarith` to compile Coq or extract it to OCaml. Only when using `Big_int_Z` do we need `zarith` to compile the extracted OCaml code, but my [implementation](https://github.com/coq/coq/pull/8252/files#diff-a0236bab39290033da13a82f61a4dd0dR726) always requires `zarith`.
Is there a way to auto-detect usage of modules, in compilation stage?

Reply:
You can provide a `coq.plugins.extraction.runtime` package, but it should not depend on `coq.ltac` for example. This meta file is indeed for compiling the plugin itself so `zarith` should not be there.

Reply:
The runtime package should contain the minimal set of dependencies that _extracted_ code need to run.